### DISCUSSION TITLE
uuidgen ???

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#776] [General]
Found `uuidgen` in `../zopen/usr/local/bin/` and noticed that there is no port in the list of repositories and there is no `man` page (s) for it.

????

**Comment by IgorTodorovskiIBM** [comment]
Ah yes, uuidgen is part of https://github.com/ZOSOpenTools/util-linuxport which we have only partially ported. We'll need to selectively enable the man pages for the tools that are made available. Can you open an issue under the util-linux port?

Btw, you can use: 
```
 zopen query --whatprovides uuidgen
```
to find out where a tool comes from

**Comment by lbdyck** [comment]
issue opened

query - not so much 

![image](https://github.com/ZOSOpenTools/meta/assets/42328411/aa50f298-a3c3-4f62-a424-7062a797e0a7)
